### ISSUE TITLE
[BUG]selenenium time out

### ISSUE THREAD

**Issue by reDpz** [charlesbel/Microsoft-Rewards-Farmer#136]
*MAKE SURE THE ERROR IS NOT FROM YOU, THIS IS THE MOST FREQUENT CASE*

**Error Log**
```
[LOGIN] Logging-in...
[LOGIN] Writing email...
[LOGIN] Writing password...
[LOGIN] Passing security checks...
[LOGIN] Logged-in !
[LOGIN] Ensuring login on Bing...
←[92m[LOGIN] Logged-in successfully !←[00m
←[92m[POINTS] You have 0 points on your account !←[00m
Traceback (most recent call last):
  File "C:\Users\redpz\Documents\Microsoft-Reward-Chrome-Ext-2.24.1.2\Microsoft-Rewards-Farmer-master\ms_rewards_farmer.py", line 778, in <module>
    waitUntilVisible(browser, By.XPATH, '//*[@id="navs"]/div/div/div/div/div[4]/a', 20)
  File "C:\Users\redpz\Documents\Microsoft-Reward-Chrome-Ext-2.24.1.2\Microsoft-Rewards-Farmer-master\ms_rewards_farmer.py", line 159, in waitUntilVisible
    WebDriverWait(browser, time_to_wait).until(ec.visibility_of_element_located((by_, selector)))
  File "C:\Users\redpz\AppData\Local\Programs\Python\Python311\Lib\site-packages\selenium\webdriver\support\wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message:
```

**Screenshot of the page**
![image](https://user-images.githubusercontent.com/71374038/210023164-71576a64-37b6-4090-b4fa-6cdfd21590e3.png)

Then the program just doesnt do anything and stays frozen like that

**Comment by reDpz** [comment]
also it says i have 0 points on my account even tho i have 81 (as it says in the dash board)

**Comment by AlexyGalaxy** [comment]
> also it says i have 0 points on my account even tho i have 81 (as it says in the dash board)

Make sure you have manually logged into your bing rewards account and clicked join microsoft rewards to show you the "Microsoft Bing Dashboard".

You also need to check your accounts file to make sure it has correct syntax and check your main .py file to make sure it has correct syntax. If you have something out of place the script will do this. Id reccomend using Visual Studios Code. Once open it should recommend you to install a python extention. Then Visual Studios Code should recomend changes to help make the script run.

Or uninstall and redownload. Here is a README I have created for this project to help. Good Luck.

README How to Install:
-Install zip from GitHub: https://github.com/charlesbel/Microsoft-Rewards-Farmer

-Install Python https://www.python.org/downloads/
Select custom installation, change nothing hit yes until installed.

-Open Microsoft-Rewards-Farmer-master, move requirements.txt to C:\Users\Alex

-Open command prompt. win + R>CMD, then type (without quotes): 'pip install -r requirements.txt' Press enter

-Intall Google Chrome.

-Install ChromeDriver from https://chromedriver.chromium.org/downloads
Place the file in X:\Windows (X as your Windows disk letter)

-Edit the accounts.json.sample with your accounts credentials and rename it by removing .sample at the end
If you want to add more than one account, the syntax is the following :
[
{
"username": "Your Email",
"password": "Your Password"
},
{
"username": "Your Email",
"password": "Your Password"
}
]

-How to Fix Mobile:
Comment out around line 134-138 with double quotes above and below
""
#Refresh page
browser.get('https://bing.com/')
#Wait 5 seconds
time.sleep(10)
#Update Counter
""

-How to run Script:
Run the script by double clicking on ms_rewards_farmer.py

**Comment by charlesbel** [comment]
Fixed with v3.0 #188
